workshops/request.html

```diff
@@ -174,14 +185,14 @@
-</label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="Non-profit: reduced fee for subsequent workshop/year (US$750)" id="group_793641257_2" role="radio" class="ss-q-radio" aria-label="Non-profit: reduced fee for subsequent workshop/year (US$750)" required="" aria-required="true"></span>
-<span class="ss-choice-label">Non-profit: reduced fee for subsequent workshop/year (US$750)</span>
-</label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="For-profit: full fee for first workshop/year (US$5000)" id="group_793641257_3" role="radio" class="ss-q-radio" aria-label="For-profit: full fee for first workshop/year (US$5000)" required="" aria-required="true"></span>
-<span class="ss-choice-label">For-profit: full fee for first workshop/year (US$5000)</span>
-</label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="For profit: full fee for subsequent workshop/year (US$3000)" id="group_793641257_4" role="radio" class="ss-q-radio" aria-label="For profit: full fee for subsequent workshop/year (US$3000)" required="" aria-required="true"></span>
-<span class="ss-choice-label">For profit: full fee for subsequent workshop/year (US$3000)</span>
+<ul class="ss-choices" role="radiogroup" aria-label="Which of the following applies to your payment for the administrative fee?  "><li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="Non-profit: full fee for first workshop/year (US$2500)" id="group_793641257_1" role="radio" class="ss-q-radio" aria-label="Non-profit: full fee for first workshop/year (US$2500)" required="" aria-required="true"></span>
+<span class="ss-choice-label">Non-profit: full fee for first workshop/year (US$2500)</span>
+</label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="Non-profit: reduced fee for subsequent workshop/year (US$1250)" id="group_793641257_2" role="radio" class="ss-q-radio" aria-label="Non-profit: reduced fee for subsequent workshop/year (US$1250)" required="" aria-required="true"></span>
+<span class="ss-choice-label">Non-profit: reduced fee for subsequent workshop/year (US$1250)</span>
+</label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="For-profit: full fee for first workshop/year (US$10,000)" id="group_793641257_3" role="radio" class="ss-q-radio" aria-label="For-profit: full fee for first workshop/year (US$10,000)" required="" aria-required="true"></span>
+<span class="ss-choice-label">For-profit: full fee for first workshop/year (US$10,000)</span>
```

Comment:
"For-profit: full fee for first workshop/year (US$10,000)" -> "For-profit: full fee for one workshop (US$10,000)"